spinalcordtoolbox/deepseg/models.py

```diff
@@ -103,6 +103,14 @@
         "description": "SC/GM multiclass segmentation on T2*-w contrast at 7T. The model was created by N.J. Laines Medina, V. Callot and A. Le Troter at CRMBM-CEMEREM Aix-Marseille University",
         "contrasts": ["t2star"],
         "default": False,
+    },
+     "model_seg_epfl_t2w_lumbar_sc": {
+        "url": [
+            "https://github.com/ivadomed/lumbar_seg_EPFL/releases/download/r20220411/model_seg_epfl_t2w_lumbar_sc.zip"
+        ],
+        "description": "Lumbar SC segmentation on T2w contrast with 3D UNet",
+        "contrasts": ["t2"],
+        "default": False,
```

Comment:
Just as a quick heads-up: Right now, there is a [separate PR open](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3541) that will add 2 new fields to each model/task:

* `url`: The URL for the project's page (e.g. a repo for the training source code, if you have one)
* `long_description`: A more in-depth description that summarizes the methods used to train the model (e.g. training datasets, architecture, etc.)

So, could you please write up that information for me? You don't have to add it to this PR or anything -- you can just share it in a reply, and then I will add it to the other PR #3541 later on. :) 

You can see examples of what these fields are supposed to look like in: https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3541/files

Reply:
Sure.
Repo URL: https://github.com/ivadomed/lumbar_seg_EPFL

long_description:
This segmentation model for T2w spinal cord segmentation uses a 3D UNet architecture, and was created with the `ivadomed` package. Training data was provided by Nawal Kinany and Dimitry Van De Ville of EPFL, with the files consisting of T2w scans (and manual spinal cord segmentations) of 11 patients.

Thanks again!

Reply:
No, there isn't. They were all healthy AFAIK.

Reply:
Thank you! That's perfect, then. :)